docs/Guides/Advertising/accountStats.md

```diff
@@ -0,0 +1,143 @@
+---
+tags: [Advertising Intelligence, adintel, account, stats, accountstats]
+---
+# Get Account Stats
+
+`Account Stats` are the total stats of all campaigns under one advertising account that has been connected to a specific Business Location in Advertising Intelligence.  
+
+> For info on Business Locations see the [Businesses guide](../Accounts.md)
+
+# Setup
+
+Create an access token with `adintel` scopes following the [Authorization guide](../../Authorization/Authorization.md).
+
+### Fetching Account Stats
```

Comment:
This page looks alright for a first slice. It clearly tells me how to complete each of the examples. I feel like it could better explain the why. Is there an example job to be done that we can describe from the partner's perspective?
That could just be rewording of these examples or it might be a standalone page like the [online toolkit](https://developers.vendasta.com/platform/ZG9jOjExNjI0NzMz-online-toolkit).

That can come in a later PR after discussion with your PMM.